core/src/main/scala/com/salesforce/op/stages/impl/selector/ModelSelector.scala

```diff
@@ -113,6 +113,7 @@ E <: Estimator[_] with OpPipelineStage2[RealNN, OPVector, Prediction]]
   protected[op] def findBestEstimator(data: Dataset[_], dag: StagesDAG, persistEveryKStages: Int = 0)
     (implicit spark: SparkSession): Unit = {
 
+    splitter.map(_.examine(data.select(labelColName).toDF()))
```

Comment:
since we don't use the result of transformation let us do `splitter.foreach(_.examine(...))`